src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx

```diff
@@ -17,32 +16,61 @@ class SettingsLayout extends React.Component {
     routes: PropTypes.array,
   };
 
+  state = {
+    navVisible: false,
+  };
+
+  componentWillMount() {
+    this.unlisten = this.props.router.listen(() => {
```

Comment:
Since the `router` needs to be passed now, we'll need to update the settingsLayout tests to take the router param

Reply:
Fixed in 619c649